{"uuid": "702f6473-b452-4883-9b40-07ed708c3cc3", "children": ["b20cd24c-2e8c-45cb-bc87-9c86270b10db", "e5974622-f44d-4e55-9e0a-0841ec881457"], "befores": [{"name": "init_driver", "status": "broken", "statusDetails": {"message": "NotADirectoryError: [WinError 267] The directory name is invalid: 'C:\\\\Users\\\\rubyk\\\\.wdm\\\\drivers\\\\geckodriver\\\\win64\\\\v0.30.0\\\\geckodriver.exe'\n", "trace": "  File \"C:\\Users\\rubyk\\Virtual Environmemts\\New Venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\rubyk\\Virtual Environmemts\\New Venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\rubyk\\Virtual Environmemts\\New Venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\rubyk\\avactisProject\\testCases\\conftest.py\", line 15, in init_driver\n    value = webdriver.Firefox(GeckoDriverManager().install())\n  File \"C:\\Users\\rubyk\\Virtual Environmemts\\New Venv\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 159, in __init__\n    firefox_profile = FirefoxProfile(firefox_profile)\n  File \"C:\\Users\\rubyk\\Virtual Environmemts\\New Venv\\lib\\site-packages\\selenium\\webdriver\\firefox\\firefox_profile.py\", line 74, in __init__\n    shutil.copytree(self.profile_dir, newprof,\n  File \"C:\\Program Files (x86)\\Python39\\lib\\shutil.py\", line 563, in copytree\n    with os.scandir(src) as itr:\n"}, "start": 1642604416306, "stop": 1642604423543}], "start": 1642604416306, "stop": 1642604423918}